### PR TITLE
Release/shinya inoue to master

### PR DESCRIPTION
**Pull request by qiaouchicago**

### CODE REVIEW

**Review by mpsolano — APPROVED** [review]
Looks good!

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -6,160 +6,80 @@ stages:
...
     - git log -1
-    - sed -i 's|version = .*|version ="'"$GDCDICTIONARY_TARGET_VERSION"'"|' plaster.toml
+    - sed -i 's|version = .*|version ="'"$CI_COMMIT_REF_NAME"'"|' plaster.toml
     - sed -i 's/is_git_tag = .*/is_git_tag = false/' plaster.toml
     - cat plaster.toml
     - bash plaster
     - git status
-    - rm setup.cfg dev-requirements.txt
+    - sed -i 's|gdcdictionary @ git+https://github.com/NCI-GDC/gdcdictionary.git@.*#egg=gdcdictionary|gdcdictionary @ git+https://github.com/NCI-GDC/gdcdictionary.git@'"$CI_COMMIT_REF_NAME"'#egg=gdcdictionary|' setup.cfg
+    - cat setup.cfg
```
[the remark this answers is not in the document]
Comment (qiaouchicago) [reply .gitlab-ci.yml:47]: It is committed and pushed to github in next step.

File: src/gdcdictionary/python/__init__.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def load_schemas_from_dir(self, directory):
             for path in glob.glob("*.yaml"):
                 schema = self.load_yaml(path)
                 schemas[path] = schema
-                resolver = RefResolver('{}#'.format(path), schema)
+                resolver = RefResolver(f'{path}#', schema)
```
[the remark this answers is not in the document]
Comment (qiaouchicago) [reply src/gdcdictionary/python/__init__.py:90]: Will add black

File: tests/schema_test.py [hunk 3]
```diff
@@ -67,7 +67,7 @@ def test_properties_enum(self):
         for node_name, node_schema in self.dictionary.schema.items():
             for prop_name, prop_schema in node_schema["properties"].items():
                 if "enum" in prop_schema:
-                    path_str = '{}->{}'.format(node_name, prop_name)
+                    path_str = f'{node_name}->{prop_name}'
```
[the remark this answers is not in the document]
Comment (qiaouchicago) [reply tests/schema_test.py:70]: will add black

**Pull request merged**